content/stranger_case/report.md

```diff
@@ -0,0 +1,175 @@
+- 'Cuisine.pdf'
+- 'voila sombre.pdf'
+- 'Paix.pdf'
+- 'Survie.pdf'
+
+All the files seem to be poems.
+
+##### Flickr information
+
+There are two pictures on the Flickr account of Eric. This first one is a picture of a typical French britain dish: a galette. Thanks to the cups we succeeded to find the restaurant where the picture was taken. The restaurant is called 'Mademoiselle Breizh' and is located in Bretagne, France. This picture was taken on the 28th of April.
+![galette](/images/stranger_case/galette.png)
+
+The second picture seems to be a picture of a phone booth and the logo of Dairy queen on the right side of the picture. We were not able to find the exact location of the phone booth but we know that this is somewhere in Canada and maybe into the Quebec area.
+
+![phone booth](/images/stranger_case/bell.png)
+  
+#### Copprethia
+
+- **Website**: [https://copprethia.fr/](https://copprethia.fr/)
+- **Field**: Healthcare
+- **Emergency phone number**: +33 7 56 95 40 16
+- **Facebook**: [https://www.facebook.com/pages/Copprethia/109926335430191/](https://www.facebook.com/pages/Copprethia/109926335430191/)
+
+##### Employees
+
+- [Jade Raynaud: CCO](#jr)
+- [Alice Blom: CTO](#ab)
+- [Raphael Mabar: COO](#rm)
+- [Ulrich Jablonowski: DRH](#uj)
+- [Faye Terni: Assistante DRH](#ft)
+- [Hortense Lutero: CLO](#hl)
+- [Alexandre Dubais](#ad)
+- [Amine Sartori: Unknown occupation](#as)
+
+#### Jade Raynaud {#jr}
+
+{{< imgWithClass "/images/stranger_case/jr.png" "Jade Raynaud" "jr.png" "" "stranger-case-pp" >}}
+
+- **Linkedin**: [https://www.linkedin.com/in/jade-raynaud-899391275/](https://www.linkedin.com/in/jade-raynaud-899391275/)
+
+She is a Formula 1 fan.
+  
+#### Alice Blom {#ab}
+
+{{< imgWithClass "/images/stranger_case/ab.png" "Alice Blom" "ab.png" "" "stranger-case-pp" >}}
+
+- **Linkedin**: [https://www.linkedin.com/in/alice-blom-6563b2257/](https://www.linkedin.com/in/alice-blom-6563b2257/)
+
+She is a cybersecurity fan.
+  
+#### Raphael Mabar {#rm}
+
+{{< imgWithClass "/images/stranger_case/tosalios.png" "Raphael Mabar" "tosalios.png" "" "stranger-case-pp" >}}
+
+- **Twitter**: [https://twitter.com/Tosalios](https://twitter.com/Tosalios)
+- **Facebook**: [https://www.facebook.com/profile.php?id=100091989057511](https://www.facebook.com/profile.php?id=100091989057511)
+- **Alias**: Tosalios
+- **Blog**: [https://tosalio.blog/](https://tosalio.blog/)
+
+His Blog has 3 snapshots on the wayback machine but nothing seems relevant except it runs WordPress, and it is associated with the username: lenainkiller0119.
+On his Twitter page we found several pieces of information: the first is that he is a fan of cycling and more particularly of the rider Thibaut PINOT. We also know that he follows the forum [https://www.jeuxvideo.com/](https://www.jeuxvideo.com/) maybe we can find something interesting there. The last big piece of information we learn is that he's a fan of the game Clash of Clans. 
```

Comment:
You can mention that we checked if his email is in use on jeuxvideo.com using "Forgot password" and we arrived at the conclusion that he doesn't have an account there.

Reply:
Ok added it.